pkgs/by-name/me/meteo-qt/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  qt5,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "meteo-qt";
+  version = "3.4";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "dglent";
+    repo = "meteo-qt";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-J9R6UGfj3vaPfn0vmjeRMsHryc/1pxoKyIE9wteVYbY=";
+  };
+
+  nativeBuildInputs = [
+    qt5.qttools
+    qt5.wrapQtAppsHook
+  ];
+
+  build-system = with python3Packages; [ sip ];
+
+  dependencies = with python3Packages; [
+    lxml
+    pyqt5
+  ];
+
+  pythonImportsCheck = [ "meteo_qt" ];
+
+  makeWrapperArgs = [ "\${qtWrapperArgs[@]}" ];
+
+  meta = {
```

Comment:
Please set either platforms = linux or platforms = unix, depending on where the program can run.